feat: Sharpen an image

viveknathani opened the pull request:
There are two ways to do this. (i) unsharp masking and (ii) convolution
Currently, convoultion is implemented since (ii) is faster than (i). In
future, (i) can be implemented too to make the software feature-rich.

viveknathani committed:
feat: Sharpen an image

There are two ways to do this. (i) unsharp masking and (ii) convolution
Currently, convoultion is implemented since (ii) is faster than (i). In
future, (i) can be implemented too to make the software feature-rich.

priyanshul-govil commented on src/secur3dit/filters/Filters.java:
```diff
@@ -502,4 +502,58 @@ public static BufferedImage pixelate(BufferedImage image, int pixelWidth) {
+     * iteration will always be 9. Hence, this kernel is not made separable and 
+     * works in O(height * width), similar to box blur with a separable kernel 
+     * and varying kernelRadius. The intensity can theoretically range from 
+     * [0, infinity]. However, in practice, after a threshold, the higher values 
```
should be [0, infinity)

priyanshul-govil reviewed (changes requested):
need change in the specified range of values in the comments.

viveknathani committed:
refactor: incorrect range of value in the comment

Range indicates "closed infinity" which does not make sense.
Change ']' to ')'.

viveknathani commented:
fixed

priyanshul-govil reviewed (approved):
seems good.
